oxidation/libparsec/crates/types/src/regex.rs

```diff
@@ -0,0 +1,35 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+use std::str::FromStr;
+
+use fancy_regex::{self, RegexBuilder};
```

Comment:
Since regex are now handled in the Rust code, do we have planned to remove this crate in favor of [`regex`](https://crates.io/crates/regex) ?

Reply:
Yes, I've planned to try to replace `fancy_regex` by `regex` since `fnmatch-regex` generates simpler regex. This PR is still in a WIP stage

Reply:
The replacement will be done in this PR ?

Reply:
Yes I'm working on it